### DISCUSSION TITLE
To be able to do multi-click with the buttons

### DISCUSSION THREAD

**Discussion by McMtech** [Blackymas/NSPanel_HA_Blueprint#1423] [Ideas]
Could we have the function to do multi-click with the buttons e.g. press once light comes on, press twice triggers a different light and third press can trigger an automation?

**Comment by edwardtfn** [comment]
Let's handle this one on #1425

**Reply by edwardtfn** [reply]
Hey @McMtech, are you still looking for an answer here?
I've posted a couple of messages on #1425.

**Comment by McMtech** [comment]
Hello Thank you I will take a look :) Kind regards Joe Fellows-McMurrayOn 18 Dec 2023, at 8:13 pm, Edward Firmo ***@***.***> wrote:﻿
Hey @McMtech, are you still looking for an answer here?
I've posted a couple of messages on #1425.
